configs/sst_security_crypto-intel.yaml

```diff
@@ -5,10 +5,12 @@ data:
   description: libraries needed for acceleration of crypto and compression on Intel processors
   maintainer: sst_security_crypto
 
-  packages:
-  - intel-ipsec-mb
-  - intel-ipp-crypto-mb
-
   labels:
   - eln
   - c9s
+
+  arch_packages:
+    x86_64:
+      - arch-specific-package1
```

Comment:
This line is a typo.

Reply:
Yikes. copy/paste error. Thanks for catching.